driver-async/src/main/com/mongodb/async/client/FindIterable.java

```diff
@@ -189,6 +189,15 @@
      */
     FindIterable<T> hint(@Nullable Bson hint);
 
+    /**
```

Comment:
Now that we have hintString, can you expand the documentation for the `hint` method to make it clear what the different is between the two, and which takes precedence if both are set?

Reply:
Done